packages/desmos-std/src/profiles/query_routes.rs

```diff
@@ -0,0 +1,47 @@
+use crate::types::PageRequest;
+use cosmwasm_std::Addr;
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub enum ProfilesRoutes {
+    Profile {
+        user: Addr,
+    },
+    IncomingDtagTransferRequests {
+        receiver: Addr,
+        pagination: Option<PageRequest>,
+    },
+    Relationships {
+        user: Addr,
+        subspace_id: u64,
```

Comment:
It should be cosmwasm_std::Uint64 instead. The u64 response from desmos is string type.